ts/torch_handler/unit_tests/run_unit_tests.sh

```diff
@@ -0,0 +1,54 @@
+#!/bin/bash
```

Comment:
As far as I understood this, it will do end to end testing of default handlers without deploying them. However, it seems this will be a manual step. We are already doing end to end tests for default handlers however with deployment via `torchserve_sanity.sh`. Is there any other advantage of adding this? If yes, then we should make it part of sanity testing.

Reply:
The main advantage is being able to quickly test changes on just the handlers without having to run the whole torchserve_sanity.sh script. I found it faster to iterate this way.

Reply:
Can we document it in  [default handler doc](https://github.com/pytorch/serve/blob/master/docs/default_handlers.md). Also, need to document batch support in default handlers [excpet text class..].

Reply:
Added to the doc.